src/javascript/app/base/header.js

```diff
@@ -169,6 +169,13 @@ const Header = (() => {
                 on_mobile: true,
 
             },
+            derivx: {
```

Comment:
derivx should be enabled only for svg countries for now

Reply:
Hi @balakrishna-binary could you please add this to the card's expected behavior or add a note for this?